### PR TITLE
feat: rollup rework

### PR DESCRIPTION
**Pull request by mpolitzer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,6 @@ The Cartesi gRPC Interfaces repository contains all gRPC and Protobuf definition
 
 - [cartesi-machine.proto](cartesi-machine.proto): contains the services exported by the cartesi machine that are consumed by the machine manager and also the definition of the lowest level messages used in multiple interfaces
 - [machine-manager.proto](machine-manager.proto): services and higher level message types used to interact with the machine manager sessions
-- [rollup-machine-manager.proto](rollup-machine-manager.proto): services and higher level message types used to interact with the rollup machine manager sessions
```
Comment (diegonehab) [inline README.md:7]: server-manager right? It's just the name that is wrong?

File: cartesi-machine.proto [hunk 2]
```diff
@@ -212,12 +212,9 @@ message HTIFConfig {
     bool yield_automatic = 5;
 }
 
-message RollupConfig {
+message CmioConfig {
```
Comment (diegonehab) [inline cartesi-machine.proto:215]: Other configs use capitals. So change to CMIOConfig.

**Review by diegonehab — CHANGES_REQUESTED** [review]
See in-line comments.

File: README.md (hunk 1 above)
Comment (mpolitzer) [reply README.md:7]: updated to `server-manager.proto`

**Comment by mpolitzer** [comment]
Repository removed from cartesi-machine.

**Pull request closed without merging**